project/Dependencies.scala

```diff
@@ -18,19 +18,18 @@ object Dependencies {
   val ScalaTestPlusJUnitVersion = ScalaTestVersion + ".0"
 
   val AwsSdkVersion = "1.11.837"
-  val JacksonVersion = "2.10.5"
-  val JacksonDatabindVersion = "2.10.5.1"
+  val JacksonVersion = "2.11.4"
 
   // often called-in transitively with insecure versions of databind / core
   private val JacksonDatabind = Seq(
-    "com.fasterxml.jackson.core" % "jackson-databind" % JacksonDatabindVersion
+    "com.fasterxml.jackson.core" % "jackson-databind" % JacksonVersion
   )
 
   private val JacksonDatatype = Seq(
     "com.fasterxml.jackson.datatype" % "jackson-datatype-guava" % JacksonVersion,
     "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % JacksonVersion,
     // Specifying guava dependency because older transitive dependency has security vulnerability
-    "com.google.guava" % "guava" % "27.1-jre"
+    "com.google.guava" % "guava" % "29.0-jre"
```

Comment:
Could we remove this now and rely on the transitive dependency instead?

Reply:
I don't think so: https://repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-guava/2.11.4/jackson-datatype-guava-2.11.4.pom still specifies 0.20

Reply:
ok, hopefully switching from 27 to 29 doesn't break something else

Reply:
`com.orbitz.consul:consul-client:1.5.3` was already bringing in version 29.0-jre, so sbt users would already get that one. I updated this one here so maven users are in sync again.